package.json

```diff
@@ -29,6 +29,7 @@
     "build": "turbo run build",
     "dev": "turbo run dev --concurrency=12",
     "test": "turbo run test",
+    "test:specific": "node scripts/testSpecific.js",
```

Comment:
`test:specific` feels too long to type. The goal of this task was to extend the `test` command itself.

Ideal usage:

```sh
# runs all tests
pnpm test

# runs all tests in css-workshop
pnpm run test --filter=css-workshop

# runs Alert tests in all workspaces
pnpm test Alert
```

Reply:
So i've been able to work through every issue with the PR besides this one. One approach I've tried has been to replace instances of `pnpm run test` with the direct command `pnpm turbo run test` using `spawn()` in my new script file. This works for most of the use cases, except for the visual tests for css and react which require multiple filters, [which should be supported by Turbo](https://turbo.build/repo/docs/core-concepts/monorepos/filtering#multiple-filters). When investing into `turbo.json` I found that filtering for the specific tests had been configured but can't locate where configuration for specific components is happening. My approach is going to be looking for into the `.config` files.

Reply:
Current solution is working for these use cases. There are some user experiences issues I've highlighted in another comment